VideoLocker/src/main/java/org/edx/mobile/view/LaunchActivity.java

```diff
@@ -83,6 +83,16 @@ public void onClick(View v) {
         fetchRegistrationDescription();
     }
 
+    protected void onResume(){
+        super.onResume();
+        PrefManager pm =new PrefManager(LaunchActivity.this, PrefManager.Pref.LOGIN);
+        //MOB-1343 : app enter here when user in the login window and lock the screen
+        if (pm.getCurrentUserProfile() != null) {
+            Intent intent = new Intent(LaunchActivity.this, MyCoursesListActivity.class);
+            startActivity(intent); 
+        }
```

Comment:
For launching any screen, we use <code>Router</code> class. Also, we use flags to avoid launching duplicate instances of the screens as duplicate instances may cause interaction issues.